src/vorta/utils.py

```diff
@@ -273,3 +274,41 @@ def is_system_tray_available():
         is_available = tray.isSystemTrayAvailable()
 
     return is_available
+
+
+def validate_passwords(firstPass, secondPass):
```

Comment:
When we implement password changing in the future this will need to be reused. Websites do password validation in real time, so it would be nice to have that here as well.

Reply:
Good call. I didn't notice it's in utils.